cloudmarker/alerts/emailalert.py

```diff
@@ -1,119 +1,41 @@
 """Email alert plugin."""
 
 
-import email
 import logging
-import smtplib
-from email.mime.multipart import MIMEMultipart
-from email.mime.text import MIMEText
+
+from cloudmarker import util
 
 # Define module-level logger.
 _log = logging.getLogger(__name__)
 
 
 class EmailAlert:
-    """A plugin to send email notification for anomalies found."""
+    """A plugin to send email alerts."""
 
-    def __init__(self, host, port, subject, to, sender, body, use_ssl=True,
-                 username=None, password=None):
+    def __init__(self, **kwargs):
```

Comment:
Since you have remove the arguments from EmailAlert plugin, it would be safe to remove `R0902`
from `pylama.ini` for emailalert.

> [pylama:cloudmarker/alerts/emailalert.py]
> ignore = R0913,C0412,R0902 //Please remove R0902

Reply:
Good point. Turns out I can remove the entire `[pylama:cloudmarker/alerts/emailalert.py]` section here. [Fixed here at line 30](https://github.com/cloudmarker/cloudmarker/pull/108/files#diff-21c09aff342003d14df85a14eeaae393L30).